src/app/public/modules/back-to-top/back-to-top.directive.ts

```diff
@@ -38,9 +51,32 @@ import {
 })
 export class SkyBackToTopDirective implements AfterViewInit, OnDestroy {
 
+  @Input()
+  public set skyBackToTop(value: SkyBackToTopOptions) {
+    if (value.hideBackToTopButton !== undefined) {
+      this.hideButton = value.hideBackToTopButton;
```

Comment:
Since this options object is only used for the back-to-top directive, I'd get rid of the `BackToTop` portion of this property name. I'd also switch to an adjective rather than a verb, since verbs are better suited for methods rather than properties. My vote is for `buttonHidden`.